tilings/strategies/assumption_insertion.py

```diff
@@ -181,3 +198,51 @@ def to_jsonable(self) -> dict:
     @classmethod
     def from_dict(cls, d: dict) -> "AddAssumptionFactory":
         return cls()
+
+
+class AddInterleavingAssumptionFactory(StrategyFactory[Tiling]):
+    def __init__(self, unions: bool = True):
```

Comment:
I think the default for unions should be false

Reply:
sure